js/impress.js

```diff
@@ -205,9 +200,7 @@
         height: 768,
         maxScale: 1,
         minScale: 0,
-
```

Comment:
Can JSCS or JSHINT handle this? It might be more useful if we could automate this kind of stuff to reduce the chances it's spotted in unrelated PRs.